content/en/docs/reference/scheduling/config.md

```diff
@@ -147,10 +182,24 @@ that are not enabled by default:
   Service across nodes.
   Extension points: `PreFilter`, `Filter`, `Score`.
   
-## Multiple profiles
+### Multiple profiles
+
+You can configure `kube-scheduler` to run more than one profile.
+Each profile has an associated scheduler name and can have a different set of
+plugins configured in its [extension points](#extension-points). For example:
+
+```yaml
+apiVersion: kubescheduler.config.k8s.io/v1beta1
+kind: KubeSchedulerConfiguration
+profiles:
+  - schedulerName: default-scheduler
+  - schedulerName: no-scoring-scheduler
+    plugins:
+      score:
+        disabled:
+        - name: '*'
+```
```

Comment:
Would it make sense to explain this example? For example by adding a paragraph:

> The example above will configure `kube-scheduler` to run two schedulers: the default one and `no-scoring-scheduler` with the `score` extension point disabled for all plugins.

Reply:
Good suggestion

Reply:
this is interesting, so are we saying we will be able to use the same scheduler binary to act as two different schedulers based on the scheduler field in the Pod ?

Reply:
Correct.